Source/MovieContainer.swift

```diff
@@ -133,7 +133,7 @@ class MovieContainer: UIView {
                                 } else if player.rate != 1.0 {
                                     player.rate = 1.0
                                 }
-                            })
+                            }) as AnyObject?
```

Comment:
Any way we can avoid this? Or at least prettify this call? Looks so very odd here. Also possible to remove the `)`?